rules_java_gapic/java_gapic_pkg.bzl

```diff
@@ -13,10 +13,17 @@
 # limitations under the License.
 
 load("@com_google_api_gax_java_properties//:dependencies.properties.bzl", "PROPERTIES")
+load("@com_google_protobuf//:protobuf_version.bzl", "PROTOBUF_JAVA_VERSION")
 
 def _wrapPropertyNamesInBraces(properties):
```

Comment:
I now feel it's strange to add protobuf within this wrapPropertyNamesInBraces function. Can you modify the dictionary before calling this method?

Reply:
done.